test/integrations/destinations/af/processor/data.ts

```diff
@@ -1838,4 +1838,449 @@ export const data = [
       },
     },
   },
+  {
+    name: 'af',
+    description: 'No authentication key is present',
+    feature: 'processor',
+    module: 'destination',
+    version: 'v0',
+    input: {
+      request: {
+        body: [
+          {
+            message: {
+              type: 'track',
+              event: 'Order Completed',
+              sentAt: '2020-08-14T05:30:30.118Z',
+              context: {
+                externalId: [{ type: 'appsflyerExternalId', id: 'afUid' }],
+                source: 'test',
+                app: { namespace: 'com.rudderlabs.javascript' },
+                os: { name: 'android' },
+                traits: { anonymousId: '50be5c78-6c3f-4b60-be84-97805a316fb1' },
+                library: { name: 'rudder-sdk-ruby-sync', version: '1.0.6' },
+              },
+              messageId: '7208bbb6-2c4e-45bb-bf5b-ad426f3593e9',
+              timestamp: '2020-08-14T05:30:30.118Z',
+              properties: {
+                tax: 2,
+                total: 27.5,
+                coupon: 'hasbros',
+                revenue: 48,
+                price: 25,
+                quantity: 2,
+              },
+              anonymousId: '50be5c78-6c3f-4b60-be84-97805a316fb1',
+              integrations: { AF: { af_uid: 'afUid' } },
+            },
+            destination: {
+              Config: {
+                androidAppId: 'com.rudderlabs.javascript',
+                groupTypeTrait: 'email',
+                groupValueTrait: 'age',
+                trackProductsOnce: false,
+                trackRevenuePerProduct: false,
+                addPropertiesAtRoot: true,
+              },
+            },
+          },
```

Comment:
Please us the new format of writing test cases its difficult to review such large test files

Reply:
Take klaviyo as reference